internal/session/job_session_cleanup_test.go

```diff
@@ -95,7 +95,7 @@ func TestSessionConnectionCleanupJob(t *testing.T) {
 
 	// Create the job.
 	job, err := newSessionConnectionCleanupJob(ctx, rw, gracePeriod)
-	job.gracePeriod = gracePeriod // by-pass factory assert so we dont have to wait so long
+	job.workerStatusGracePeriod = gracePeriod // by-pass factory assert so we dont have to wait so long
```

Comment:
This is also needed on the `main` branch

Reply:
I would think the other change (not swagger) is as well. Maybe make this against `main` and then backport?

Reply:
Looks like those changes are part of this PR to `main`: https://github.com/hashicorp/boundary/pull/5084.